maven/bnd-resolver-maven-plugin/src/main/java/aQute/bnd/maven/resolver/plugin/ResolverMojo.java

```diff
@@ -0,0 +1,84 @@
+package aQute.bnd.maven.resolver.plugin;
```

Comment:
This plugin will not run during a normal build. It is mostly for command line use to mutate a bndrun file's -runbundles. So I don't think we need an m2e mapping.